src/filter/json_encode_filter.rs

```diff
@@ -3,9 +3,10 @@
 //! This filter takes `LogLines` and encodes them into JSON, emitting the
 //! encoded event as a Raw event. This allows further filters or sinks to
 //! operate on the JSON without needing to understand a `LogLine` event in
-//! particular.  If the ``LogLine`` value is a valid JSON object and `parse_line`
-//! config option is true, then the JSON will be merged with `LogLine`
-//! metadata. Otherwise, the original line will be included simply as a string.
+//! particular.  If the ``LogLine`` value is a valid JSON object and
```

Comment:
'``LogLine``' should be '`LogLine`'.

99% sure that's my search-and-replace goof.

Reply:
Eh?

Reply:
Oh I see now when Editing your message. Change double backtick to single backtick.

Reply:
Heh, markdown is hard.